R/helper.R

```diff
@@ -98,19 +96,58 @@ run_app <- function(app_name, qc_dir, lib_path, config_path) {
     url <- sprintf("http://127.0.0.1:%s", port)
 
 
+    new_rstudioapi <- {
+      version_diff <- compareVersion(packageDescription("rstudioapi", fields = "Version"), "0.16-0")
+      if (version_diff < 0) {
+        FALSE
+      }
+      else {
+        TRUE
+      }
+    }
```

Comment:
Curious why you wrote it this way. This line does the same thing, and I think cleaner. Also maybe move the comment on 125 to up here?

`new_rstudioapi <- compareVersion(packageDescription("rstudioapi`, fields = "Version"), "0.16.0") != -1`

Reply:
Or actually, keep the comment on 125 there and add a new comment here to explain what this is doing